src/azure-cli-core/azure/cli/core/tests/test_argcomplete.py

```diff
@@ -0,0 +1,65 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from unittest import mock
+
+from azure.cli.core.util import run_cmd
+from azure.cli.testsdk import ScenarioTest
+
+
+class ArgcompleteScenarioTest(ScenarioTest):
```

Comment:
There is no `ScenarioTest` in `azure-cli-core`. I suggest changing the test to `unittest.TestCase`, such as

https://github.com/Azure/azure-cli/blob/d1a5844544ba30da428505867ad2f1ea39f2b950/src/azure-cli-core/azure/cli/core/tests/test_profile.py